README.md

```diff
@@ -530,6 +530,114 @@ Terraform module to automate your infrastructure provisioning. See [examples](ht
     Note: It can take **up to 5 minutes** from when you configure the Workload
     Identity Pool mapping until the permissions are available.
 
+
+## Troubleshooting
```

Comment:
What do you think about having a separate troubleshooting doc? The main readme is pretty long and I wonder if this would be hard to discover.

Reply:
Moved to a different page